server/websocket.go

```diff
@@ -579,8 +579,13 @@ func (p *Plugin) WebSocketMessageHasBeenPosted(connID, userID string, req *model
 			p.LogError("missing channelID")
 			return
 		}
+		title, ok := req.Data["title"].(string)
+		if !ok {
+			p.LogError("missing title")
+			return
+		}
```

Comment:
Does this make it a requirement for a title to be there? I think we should still continue if it's missing.

Reply:
I see, we are sending an empty string either way. I guess we could make it completely optional to avoid even sending the field but it's okay really.

Reply:
I think this is now preventing us from starting a call without a message given we made it an optional parameter on the client. I think it's safe to remove the check completely given this is optional. Otherwise only check if the `title` field exists and it's empty at the same time, but again that may require client changes.

Reply:
Done